### DISCUSSION TITLE
phenoage和phenoage_advance不匹配

### DISCUSSION THREAD

**Discussion by lbj11111** [dayoonkwon/BioAge#37] [General]
很感谢您能开发这样一款计算生物年龄的代码，我想请教您一个问题就是当我用：
phenoage_data_OA2 = phenoage_calc(data =data_OA2,
                                 biomarkers = c("albumin_gL","alp","crp","creat_umol","glucose_mmol","wbc","lymph","rbc","mcv"),
                                 fit = phenoage$fit)代码算出来后，调整好参数后任然有一些inf，且主要问题在于我的phenoage和phenoage_advance是不匹配的，想问为什么会出现这样的情况？

**Comment by dayoonkwon** [comment]
Please post in English. See Discussion #3 for Inf values.
